compiler/rustc_attr/src/session_diagnostics.rs

```diff
@@ -200,15 +202,16 @@ pub(crate) struct InvalidReprHintNoValue {
 }
 
 // Error code: E0565
-pub(crate) struct UnsupportedLiteral {
+pub(crate) struct UnsupportedLiteral<'a> {
     pub span: Span,
     pub reason: UnsupportedLiteralReason,
     pub is_bytestr: bool,
+    pub source_map: &'a SourceMap,
```

Comment:
Is it possible to precompute the start point span when constructing `UnsupportedLiteral`?

Reply:
Addressed here https://github.com/rust-lang/rust/pull/101434/commits/dd5850b8faf04e1b98542cf0813920385a6cc4db.
Thank you!